api/views.py

```diff
@@ -17,7 +17,10 @@ def post(self, request):
         if serializer.is_valid():
             order = serializer.validated_data
 
-            Client(order).create_delivery()
+            delivery = Client(order).create_delivery()
+
+            if 'error' not in delivery.to_dict():
+                delivery.print_pdf()
```

Comment:
again, this is just to get a PDF file out and see if its contents look ok. It's by no means what we'll use in production.